Shoko.Server/API/v3/Controllers/SeriesController.cs

```diff
@@ -134,6 +134,22 @@ public ActionResult<List<SeriesRelation>> GetShokoRelationsBySeriesID([FromRoute
                 .ToList();
         }
         
+        /// <summary>
+        /// Get a paginated list of <see cref="Series"/> without local files, available to the current <see cref="User"/>.
+        /// </summary>
+        /// <param name="pageSize">The page size.</param>
+        /// <param name="page">The page index.</param>
+        /// <returns></returns>
+        [HttpGet("WithoutFiles")]
+        public ActionResult<ListResult<Series>> GetSeriesWithoutFiles([FromQuery] [Range(0, 100)] int pageSize = 50, [FromQuery] [Range(1, int.MaxValue)] int page = 1)
+        {
+            var user = User;

```

Comment:
What is the use of this temporary variable? Shouldn't this have been in-lined for the one usage?